circle.yml

```diff
@@ -6,6 +6,8 @@ machine:
 
 dependencies:
   override:
+    - docker version
+    - docker build -t app:build -f Dockerfile.deploy .
```

Comment:
lol, I didn't know about `-t` option for `build` and always wrote some wrapper scripts that got the image-id from the output. I like it 😆